modules/suncalc.js

```diff
@@ -335,7 +335,7 @@ SunCalc.getMoonTimes = function (date, lat, lng, inUTC) {
 
 // export as Node module / AMD module / browser variable
 if (typeof exports === 'object' && typeof module !== 'undefined') module.exports = SunCalc;
-else if (typeof define === 'function' && define.amd) define(SunCalc);
-else window.SunCalc = SunCalc;
+//else if (typeof define === 'function' && define.amd) define(SunCalc);
+//else window.SunCalc = SunCalc;
```

Comment:
For these two, banglejs won't be using them but I wonder if they were intended so this module can run elsewhere - I guess not since @gfwilliams it's your change?

Reply:
Thats from the source module:
https://github.com/mourner/suncalc/blob/master/suncalc.js#L313

Reply:
Yeah, I was wondering that. As it's an external module the less changes we can make the better really.

I'd be tempted to just leave it and add a comment to exclude it from the linter? I can definitely imagine we might end up with a case where an interface.html file wants to use it for something, so it'd still need to be able to be run in the browser at minimum.

Reply:
> so it'd still need to be able to be run in the browser at minimum.

The browser would be supported with the `globalThis` solution. Then everything works except amd modules, but I don't think there is anything in the Banglejs ecosystem that would use AMD modules?

Reply:
Yeah I don't think so - I guess it's a question of whether we keep the module exactly the same and treat it as an external unlinted dependency, perhaps there are some others like this too?

Reply:
I have merged the solution that I like best. Now suncalc will be exported via the CommonJS module if it exists, and otherwise it is set on `globalThis` which is generally speaking universally supported. No eslint comments are needed for the code to pass lint.

Let me know if you still prefer no changes and eslint comments.

Reply:
Staying as close as possible to upstream makes it easier to apply patches, but this one doesn't look to maintained.

Reply:
Yeah I agree, I think we merge this - shall let @gfwilliams make the call though, in case he had any further thoughts